src/pool-bin/BinPoolManager.sol

```diff
@@ -66,10 +66,10 @@ contract BinPoolManager is IBinPoolManager, ProtocolFees, Extsload {
         external
         view
         override
-        returns (uint128 binReserveX, uint128 binReserveY, uint256 binLiquidity)
+        returns (uint128 binReserveX, uint128 binReserveY, uint256 binLiquidity, uint256 binShare)
```

Comment:
is it better to call it or `totalShare` 😂

No strong opinion on this feel free if u prefer current naming

Reply:
i was debating on this too 😆, if we set `totalShare` here, would people start to ask about `binLiquidity -> totalLiquidity, binReserveX -> totalReserveX` etc..?

Reply:
updated